helm.tf

```diff
@@ -34,21 +8,30 @@ resource "helm_release" "ingress_nginx" {
   create_namespace  = true
   repository        = "https://kubernetes.github.io/ingress-nginx"
   chart             = "ingress-nginx"
-  version           = "3.34.0"
+  version           = "4.0.18"
```

Comment:
We should allow chart version selection

Reply:
I'm marking this as resolved because we agreed to don't do it